pkg/driver/driver.go

```diff
@@ -157,6 +170,36 @@ func (driver *Driver) Stop(nodeService bool) error {
 	return nil
 }
 
+// StartScrubber starts the scrubber period task
+func (driver *Driver) StartScrubber(nodeService bool, scrubberInterval int64) error {
+	if nodeService {
+		log.Tracef("Scheduling scrubber task to run every %v seconds", scrubberInterval)
+		tick := time.NewTicker(time.Duration(scrubberInterval) * time.Second)
+		done := make(chan bool)
+		go func() {
+			// Cleanup ephemeral orphan volumes
+			driver.ScrubberTask(time.Now())
+			for {
+				select {
+				case t := <-tick.C:
+					driver.ScrubberTask(t)
+				case <-done:
```

Comment:
Where is this channel closed? on stop you are missing it?

Reply:
Yes, it got missed. I will fix this.